### DISCUSSION TITLE
Blog comment on GOMEMLIMIT is a game changer for high-memory applications

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1217] [Docs/blog comments]
# Blog comment on GOMEMLIMIT is a game changer for high-memory applications

Go 1.19 introduced GOMEMLIMIT, which completely changes how you can manage memory limits in Go. Learn how it helps Weaviate be more reliable.

https://weaviate.io/blog/gomemlimit-a-game-changer-for-high-memory-applications

**Comment by BitBistro** [comment]
Great article! I also checked out the Twitch article about the ballast—thanks for pointing it out. I’d be interested in learning more about common pitfalls that lead to accidental heap allocations.

**Comment by miparnisari** [comment]
I would also be interested in an article about accidental heap allocs :)

**Comment by dramallamabonanza** [comment]
FYI, the typo GOMEMLIMT (missing the second i) appears 5 times on this page.

**Comment by Archandan** [comment]
Very informative. Thank you for sharing

**Comment by dilip90** [comment]
Thanks for sharing , will check the efficiency of GOMEMLIMIT .
